packages/pn-personafisica-webapp/src/api/delegations/__test__/Delegations.api.test.ts

```diff
@@ -2,34 +2,25 @@ import MockAdapter from 'axios-mock-adapter';
 import { apiClient } from '../../axios';
 import { mockAuthentication } from '../../../redux/auth/__test__/reducers.test';
 import { DelegationsApi } from '../Delegations.api';
+import { Delegation } from '../../../redux/delegation/types';
 
-const mockDelegation = {
```

Comment:
Spostare i mock in un file test-utils.ts (prendere spunto dai test delle notifiche)